web/src/main/webapp/libs/app.bundle.min.js

```diff
@@ -116808,6 +116818,65 @@
                 })
             }
         })),
+        InProgresstList = Object(f.f)(Object(T.f)((function(e) { // BAO
+            return {
+                _inProgressList: e["features/inprogress-list"]
+            }
+        }))(class extends Pa {
+            constructor(e) {
+                super(e), this._getRenderListEmptyComponent = this._getRenderListEmptyComponent.bind(this), this._onPress = this._onPress.bind(this)
+            }
+            componentDidMount() {
+                super.componentDidMount(),
+                this._importJson();
+                if (interfaceConfig.IN_PROGRESS_LIST_INTERVAL > 0) {
+                    this._interval = setInterval(() => this._importJson(), interfaceConfig.IN_PROGRESS_LIST_INTERVAL * 1000)
+                }
+            }
+            componentWillUnmount() {
+                this._interval && (clearInterval(this._interval), this._interval = void 0)
+            }
+            _importJson() {
+                fetch("inProgressList.json")
+                    .then(res => res.ok && res.json())
+                    .then(data => this.props.dispatch({type: "UPDATE_INPROGRESS_LIST", list: data}))
+                    .catch(error => console.error(error))
+            }
+            _getRenderListEmptyComponent() {
+                const {
+                    t: e
+                } = this.props;
+                return v.a.createElement(qn.Container, {
+                    className: "meetings-list-empty",
+                    style: Ma.emptyListContainer
+                }, v.a.createElement(qn.Text, {
+                    className: "description",
+                    style: Ma.emptyListText
+                }, e("welcomepage.inProgressListEmpty", {ns: "ofmeet"})))
+            }
+            render() {
+                if (!InProgressListEnabled()) return null;
+                const {
+                    disabled: e,
+                    _inProgressList: t
+                } = this.props, n = function(e) {
+                    return e.reverse().map(e => ({
+                        date: e.date,
+                        duration: e.duration,
+                        time: [e.date],
+                        title: e.name,
+                        url: e.url
+                    }))
```

Comment:
I assign "e" to the disabled of Props, "t" to _inProgressList, and "n" to the objects that I sorted based on _inProgressList using IIFE.
I don't know why it uses IIFE, but it follows RecentList.
Each of properties (date, duration, time, title, url) of n will be used in the MeetingList component.